pkg/che/che_test.go

```diff
@@ -584,7 +584,7 @@ func cheRoute(tls bool) *routev1.Route { //nolint: unparam
 		},
 		Spec: routev1.RouteSpec{
 			Host: fmt.Sprintf("codeready-codeready-workspaces-operator.%s", test.MemberClusterName),
-			Path: "",
+			Path: "/",
```

Comment:
Can we test both cases?
Just in case the / will disappear again in the future.

Alternatively we could test the route until directly if it's easier to test.

Reply:
yes, would be great to have both cases covered (with & without path)